src/tensor_ops/gelu/cpu_kernel.rs

```diff
@@ -0,0 +1,33 @@
+use crate::tensor_ops::cpu_kernels::UnaryDerivative;
+use no_std_compat::f32::consts::PI;
```

Comment:
```suggestion
use std::f32::consts::PI;
```
since no_std_compat should overwrite the std namespace